components/tag-connect/TC2030-IDC-NL.stanza

```diff
@@ -0,0 +1,46 @@
+#use-added-syntax(jitx)
+defpackage ocdb/tag-connect/TC2030-IDC-NL :
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+
+  import ocdb/land-patterns
+  import ocdb/symbols
+  import ocdb/box-symbol
+  import ocdb/bundles
+  import ocdb/generator-utils
+  import ocdb/generic-components
+
+; NOTE PINOUT ORDER DIFFERENT THAN 10-PIN
+; Datasheet showing pinout: https://www.tag-connect.com/wp-content/uploads/bsk-pdf-manager/2019/12/TC2030-IDC-NL-Datasheet-Rev-B.pdf
+
+pcb-landpattern footprint :
+  ; landing pads
+  for (l in grid-locs(2, 3, 1.27, 1.27), i in [2 4 6 1 3 5]) do :
+    pad p[i] : testpoint-pad(0.787) at l
+  
+  ; alignment holes
+  layer(Cutout()) = Circle((- 2.54), 0.0, 0.4955)
+  layer(Cutout()) = Circle(2.54, 1.016, 0.4955)
+  layer(Cutout()) = Circle(2.54, (- 1.016), 0.4955)
+
+  layer(Courtyard(Top)) = Rectangle(6.08, 3.032 ) ;, loc((-0.692), 0.0))
```

Comment:
Do you want to use the loc in the rectangle? If not, please delete it

Reply:
deleted